src/tag.ts

```diff
@@ -1,12 +1,12 @@
 'use strict';
 import { createListOfSqlParams } from '@src/util';
-import { SQLTemplate, Query } from './types';
+import { SQLTemplate, Query, QueryArg } from './types';
 import zip from 'lodash.zip';
 
-const getUniqueValues = (array: any[]) => (Array.isArray(array) ? [...new Set(array)] : array);
+const getUniqueValues = (value: any) => (Array.isArray(value) ? [...new Set(value)] : value);
 
 const argumentToParameters = (arg: any): string => {
-  if (!arg) {
+  if (arg === undefined) {
```

Comment:
This was the error. We would return '' for all falsy values, when we really on want undefined to return ''